pkgs/development/python-modules/wxpython/4.2.nix

```diff
@@ -61,6 +67,15 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = ''
+    cp ${waf_2_0_25} bin/waf-2.0.25
```

Comment:
Suggestion: Link [the issue you filed upstream](https://github.com/wxWidgets/Phoenix/issues/2575) here for discoverability.